smriprep/workflows/base.py

```diff
@@ -68,45 +68,45 @@ def init_smriprep_wf(
                 output_dir='.',
                 run_uuid='testrun',
                 skull_strip_fixed_seed=False,
-                skull_strip_template=('OASIS30ANTs', {}),
+                skull_strip_template=Space.from_string('OASIS30ANTs')[0],
```

Comment:
```suggestion
                skull_strip_template=Space.from_string('OASIS30ANTs').name,
```

Reply:
`skull_strip_template` is now one `Space` object. Because `Space.from_string` returns lists, the first item is selected.

Reply:
that's right, i forgot it returns a list - sorry for the noise